.devcontainer/Dockerfile.base

```diff
@@ -0,0 +1,42 @@
+FROM runpod/pytorch:2.4.0-py3.11-cuda12.4.1-devel-ubuntu22.04
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    wget \
+    nano \
+    socat \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Miniconda (optional, if you want it ready at build-time)
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+   && bash /tmp/miniconda.sh -b -p /miniconda3 \
+   && rm /tmp/miniconda.sh
+
+ENV PATH="/miniconda3/bin:${PATH}"
+RUN eval "$(/miniconda3/bin/conda shell.bash hook)"
+RUN conda create -n comfyui python=3.11 -y
+RUN conda create -n comfystream python=3.11 -y
+COPY . /app
+
+RUN git clone https://github.com/comfyanonymous/ComfyUI.git /ComfyUI
+COPY ./workflows/ui* /ComfyUI/user/default/workflows/examples
+COPY ./nodes/tensor_utils /ComfyUI/custom_nodes/tensor_utils
+
+WORKDIR /ComfyUI
+RUN conda run -n comfyui pip install -r requirements.txt
+RUN conda run -n comfyui pip install --upgrade tensorrt-cu12-bindings==10.7.0 tensorrt-cu12-libs==10.7.0
+
+# Install custom nodes
+WORKDIR /app
+RUN conda run -n comfyui python src/comfystream/scripts/setup_nodes.py --workspace /ComfyUI
+
+#Install ComfyStream requirements
+RUN conda run -n comfystream pip install tensorrt-cu12-bindings==10.7.0 tensorrt-cu12-libs==10.7.0
+RUN conda run -n comfystream pip install -r requirements.txt
+RUN conda run -n comfystream pip install .
+RUN conda run -n comfystream python install.py --workspace /ComfyUI
+RUN rm -rf /app
```

Comment:
Can you explain the logic behind this?

Reply:
After thinking further about this as I'm using it for the `ai-worker` image on https://github.com/livepeer/ai-worker/pull/405, I think this `rm` command should be removed. It will already allow the `ai-worker` repo to use the docker image to run the `setup_models.py` script. WDYT?

Reply:
Interesting work! Yes, let's keep the files. I have implemented your suggestion to copy only needed files (pending push). 

I initially thought we would just clone comfystream as usual and call `setup_nodes.py` and `setup_models.py` to install them since it is re-usable, however this is innovative. The only consideration I can think of is ai-runner current uses pyenv and this base will contain conda environments.

Reply:
This is updated now so all of `/comfystream` is copied into the base container.  The devcontainer workspace path is also `/comfystream` so that when a devcontainer is used, the base layer's comfystream archive is replaced with the current version from the git workspace